Guesser/AbstractTypeGuesser.php

```diff
@@ -11,9 +11,9 @@
 
 namespace Sonata\PropelAdminBundle\Guesser;
 
+use Propel\Bundle\PropelBundle\Form\PropelTypeGuesser;
 use Sonata\AdminBundle\Guesser\TypeGuesserInterface;
 use Sonata\AdminBundle\Model\ModelManagerInterface;
-use Symfony\Bridge\Propel1\Form\PropelTypeGuesser;
```

Comment:
This PR remove prope1-bridge which it uses on master with PropelTypeGuesser. It means this bundle cannot work with SF project >= 2.7. It means also we should drop SF obsolete versions.